docs/src/components/frame/frame.module.css

```diff
@@ -33,10 +33,28 @@
 
 .dark {
   background: var(--psColorsBackgroundDark1);
+
+  --appColorsBackgroundLo: var(--psColorsBackgroundDark1);
+  --appColorsBackgroundMed: var(--psColorsBackgroundDark2);
+  --appColorsBackgroundHi: var(--psColorsBackgroundDark3);
+  --appColorsBorderLow: var(--psColorsBorderLowOnDark);
+  --appColorsBorderHigh: var(--psColorsBorderHighOnDark);
+  --appColorsPrimaryActionText: var(--psColorsPrimaryActionTextDarkTheme);
+  --appColorsTextIconHigh: var(--psColorsTextIconHighOnDark);
+  --appColorsTextIconLow: var(--psColorsTextIconLowOnDark);
 }
 
 .light {
   background: var(--psColorsWhite);
+
+  --appColorsBackgroundLo: var(--psColorsBackgroundLight3);
+  --appColorsBackgroundMed: var(--psColorsBackgroundLight2);
+  --appColorsBackgroundHi: var(--psColorsBackgroundLight1);
```

Comment:
Is this the right order? Seemed reversed but I'm not sure.

Reply:
That is a great observation.  I think it at first seems like it's in the wrong order because the dark and the light backgrounds are listed 1, 2, 3 in opposite order from each other. In dark, 1 is most dark. In light, 3 is most light. I think the "most" contrast should be at the same extreme, either 1 or 3, matched in dark and light. To try to be more clear, for the app-level vars, I used lo, med, hi to indicate contrast away from the dark/light, depending.

Reply:
As far as naming conventions, the `-appColorsBackground` variables use abbreviations: Lo, Med, Hi, where the other variables use the full words: Low, High. Should we use the same convention?